package/src/Slider.tsx

```diff
@@ -281,6 +281,18 @@ const SliderComponent = (
           default: constants.LIMIT_MAX_VALUE,
         });
 
+  const limitCheck = useCallback(() => {
+    if (lowerLimit >= upperLimit) {
+      console.warn(
+        'Invalid configuration: lower limit is supposed to me smaller than upper limit',
+      );
+    }
+  }, [lowerLimit, upperLimit]);
+
+  useEffect(() => {
+    limitCheck();
```

Comment:
We don't need to create separate callback for `limitCheck` and use it only in the effect.
Let's have that single `if` check and `console.warn` directly in this effect.
Also, if the callback uses `lowerLimit, upperLimit` for dependencies then I don't see a reason for using anything different in the dependencies of this effect.